eo-maven-plugin/src/test/resources/org/eolang/maven/binarize/eo_env/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "eo_env"
```

Comment:
@levBagryansky same here, the underscore is not what we love in Java world

Reply:
@yegor256 This is the cargo package. They prefer snake_case like [here](https://doc.rust-lang.org/cargo/getting-started/first-steps.html) according to [naming conventions](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html)